hi-ml-azure/src/health/azure/azure_util.py

```diff
@@ -523,3 +550,45 @@ def get_aml_runs(args: Namespace, workspace: Workspace, run_id_source: AzureRunI
     else:
         raise ValueError(f"Unrecognised RunIdSource: {run_id_source}")
     return [run for run in runs if run is not None]
+
+
+def get_run_file_names(run: Run, prefix: str = "") -> List[str]:
+    """
+    Get the remote path to all files for a given Run which optionally start with a given prefix
+
+    :param run: The AML Run to look up associated files for
+    :param prefix: The optional prefix to filter Run files by
+    :return: A list of paths within the Run's container
+    """
+    all_files = run.get_file_names()
+    return [f for f in all_files if f.startswith(prefix)] if prefix else all_files
+
+
+def download_run_files(run: Run, output_dir: Path, prefix: str = "") -> None:
+    """
+    Download all files for a given run, which optionally start with a given prefix
```

Comment:
For both download methods, we should discuss how they are parameterized. At present, most people seem to have trouble instantiating the Run object in the first place.